src/app/qgsrasterlayerproperties.cpp

```diff
@@ -2043,6 +2051,27 @@ bool QgsRasterLayerProperties::rasterIsMultiBandColor()
   return mRasterLayer && nullptr != dynamic_cast<QgsMultiBandColorRenderer *>( mRasterLayer->renderer() );
 }
 
+void QgsRasterLayerProperties::updateInformationContent()
+{
+  // We are using QgsWebView and the renderer is completely different from QTextBrowser
+  // add some extra style here
+  QString myStyle = QgsApplication::reportStyleSheet();
```

Comment:
What do you think about making adding an enum argument to reportStyleSheet to control whether QTextBrowser or QWebView stylesheet is returned?

Reply:
Sounds like a good idea.

Reply:
See https://github.com/qgis/QGIS/pull/32269/commits/30eea3dfca534a82a7629fe63ac0dccdcb3d78ff